sky/utils/log_utils.py

```diff
@@ -76,14 +76,15 @@ class RayUpLineProcessor(LineProcessor):
     class ProvisionStatus(enum.Enum):
         LAUNCH = 0
         RUNTIME_SETUP = 1
+        PULLING_DOCKER_IMAGES = 2
 
     def __enter__(self):
         self.state = self.ProvisionStatus.LAUNCH
         self.status_display = safe_rich_status('[bold cyan]Launching')
         self.status_display.start()
 
     def process_line(self, log_line):
-        if ('Shared connection to' in log_line and
+        if ('Success.' in log_line and
```

Comment:
Any reason we updated this?

Reply:
Yes, the `Shared connection to` not always showed up in my test and it sometimes stuck at the `LAUNCH` state. There are two choices here: the `Success.` output by `updater.py`, or the `Got remote shell` output by `log_timer.py`.

```log
2023-07-12 16:32:20,102 SUCC updater.py:280 -- Success.
2023-07-12 16:32:20,102 INFO log_timer.py:30 -- NodeUpdater: ray-sky-e922-memory-head-76c3a3dc-compute: Got remote shell  [LogTimer=47191ms]
```

Reply:
Not sure if we need to create a new PR to do this